spec/text-objects-spec.coffee

```diff
@@ -164,28 +164,232 @@ describe "TextObjects", ->
       expect(editorElement.classList.contains('normal-mode')).toBe(true)
 
   describe "the 'it' text object", ->
+
     beforeEach ->
-      editor.setText("<something>here</something><again>")
-      editor.setCursorScreenPosition([0, 5])
+      editor.setText('clean')
+      editor.setCursorBufferPosition([0, 0])
+      keydown('y')
+      keydown('i')
+      keydown('w')
+      expect(vimState.getRegister('"').text).toBe('clean')
```

Comment:
I don't understand this...?  Looks like you're testing yanking inside a word before each inner tag test?